src/wpool_process.erl

```diff
@@ -46,9 +46,26 @@
 
 %% api
 -export([start_link/4, call/3, cast/2, send_request/2]).
+
+-ifdef(TEST).
+
+-export([get_state/1]).
+
+-endif.
+
 %% gen_server callbacks
 -export([init/1, terminate/2, code_change/3, handle_call/3, handle_cast/2, handle_info/2,
-         handle_continue/2, format_status/2]).
+         handle_continue/2]).
+
+-if(?OTP_RELEASE < 25).
+
+-export([format_status/2]).
+
+- else .
+
+-export([format_status/1]).
+
+-endif.
```

Comment:
We don't do things like this in this project, @paulo-ferraz-oliveira … We just move forward and set `min_otp_vsn` to whatever we need it to be. If people want to use worker pool with older (now unsupported in `main`) versions of Erlang they can always use a previous release. And if, like I'm pretty sure is the case with MongooseIM, we need to backport changes so they work with previous versions, we just create new minor releases for those things. Check, for instance, what happened with the `rel-3.1` branch.

Reply:
No prob. I asked [here](https://github.com/inaka/worker_pool/issues/204#issuecomment-2220819969), but you must not've seen it. So I went with the safe route.

Reply:
You're correct. I haven't seen it.